openapi.yml

```diff
@@ -0,0 +1,244 @@
+openapi: '3.0.0'
+info:
+  title: kytos/of_flow_manager
+  version: 0.1.3
+  description: >-
+    **Warning**: *This documentation is experimental and may change soon.*
+
+
+    Manages flows for OpenFlow switches, for versions 1.0 and 1.3. Can add, list
+    and remove flows from datapaths. Currently supports a subset of available
+    matches and actions from the OpenFlow specification.
+tags:
+  - name: List
+  - name: Add
+  - name: Delete
```

Comment:
I guess we can remove this tags and use only one tag named Current and change the endpoints to use the methods: ['GET', 'POST', 'DELETE']. 

IMHO is wrong to use a POST method to delete information.( http://flask.pocoo.org/snippets/1/)

Reply:
@diraol and I do think this should be a DELETE endpoint. However, we cannot send a JSON body with DELETE (https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE), and the flows are being removed according to their attributes, sent via JSON.